### DISCUSSION TITLE
cannot run alias command

### DISCUSSION THREAD

**Discussion by jobinnthomas** [cmderdev/cmder#2894] [Q&A]
alias xreplay=C:\Program Files (x86)\Program.exe

The above is not working as i get "\Program.exe was unexpected at this time"

How do i fix it?
Can you also point out why its not working and what you did to fix it?
Please just don't post the command without any explanation.

**Comment by DRSDavidSoft** [comment]
I feel like the issue might be related to the spaces in the path.
Tell you what, a better alternative might be to just add the `C:\Program Files (x86)\LeCroy\XReplay\` to your `%PATH%` environment variable, if you don't mind that this solution as it will basically open `xreplay.exe` from Cmder, Cmd.exe, Run menu, start menu, etc

**We have a guide here:**
https://github.com/cmderdev/cmder/wiki/Setting-up-Environment-Variables

No need to define the `%cmder_root%` or add it to `%PATH%`, only add the path of `XReplay`'s root directory there.

If you don't like this solution, tell me and I'll investigate the original `alias` issue further so it will work, provided that I have time.

**Reply by jobinnthomas** [reply]
I got it working by using "C:\PROGRA~2" but I was just surprised that there was no way to add quotes to "C:\Program Files (x86)\LeCroy\XReplay\Xreplay.exe" to make it work.

**Reply by daxgames** [reply]
You can use quotes.  Look at the wiki for advanced macro creation.
